catroid/gradle/standalone_apk_tasks.gradle

```diff
@@ -182,23 +182,40 @@ def removeUnecessaryPermissions() {
 }
 
 def copyScreenshotAsIcon() {
-    File screenshot = file(project.ext.standaloneTmpPath + "manual_screenshot.png")
-    if(!screenshot.exists()){
-        screenshot = file(project.ext.standaloneTmpPath + "automatic_screenshot.png")
-    }
-    copy {
-        from screenshot.getPath()
-        into project.ext.resPath
-        rename { String fileName ->
-            fileName.replace(screenshot.getName(), 'icon.png')
+    def automaticScreenshots = []
+    File screenshot
+    File standaloneTmpPath = file(project.ext.standaloneTmpPath)
+    standaloneTmpPath.traverse(type: groovy.io.FileType.ANY) {
+        if (it.getName().equals("automatic_screenshot.png")) {
+            automaticScreenshots.add(it)
+
+            println "Found automatic screenshot: " + it.getPath()
+        } else if (it.getName().equals("manual_screenshot.png")) {
+            screenshot = it
+
+            println "Found manual screenshot: " + it.getPath()
         }
     }
 
-    copy {
-        from screenshot.getPath();
-        into project.ext.resPath
-        rename { String fileName ->
-            fileName.replace(screenshot.getName(), 'splash_screen.png')
+    if (screenshot != null && !automaticScreenshots.isEmpty() && automaticScreenshots.get(0).exists()) {
```

Comment:
This is just a guess. Should it be `screenshot == null`?
When there is no manual screenshot (`screenshot == null`), an automatic one will be chosen. That's what this if clause should do, right?